src/eCRC_qaautomation/automationtest/moduletests/successpage_spec.js

```diff
@@ -0,0 +1,262 @@
+    bcServicesCardLoginPage.password.sendKeys(testInput.bcServicesCardPassword);
+    bcServicesCardLoginPage.continueButton.click();
+    bcServicesCardLoginPage.continueButton.click();
+
+    browser.wait(
+      browserWait.textToBePresentInElementValue(
+        applicationFormPage.lastName,
+        testInput.applicationFormLastName
+      ),
+      5000
+    );
+
+    expect(applicationFormPage.lastName.getAttribute("value")).toBe(
+      testInput.applicationFormLastName
+    );
+
+    expect(applicationFormPage.sex.getAttribute("value")).toBe(
+      testInput.applicationFormGender
+    );
+
+    expect(applicationFormPage.middleName.getAttribute("value")).toBe(
+      testInput.applicationFormMiddleName
+    );
+
+    expect(applicationFormPage.dateOfBirth.getAttribute("value")).toBe(
+      testInput.applicationFormDateOfBirth
+    );
+
+    expect(applicationFormPage.currentAddresCountry.getAttribute("value")).toBe(
+      testInput.applicationFormCurrentAddresCountry
+    );
+
+    expect(applicationFormPage.currentAddressCity.getAttribute("value")).toBe(
+      testInput.applicationFormCurrentAddressCity
+    );
+
+    expect(
+      applicationFormPage.currentAddresPostalCode.getAttribute("value")
+    ).toBe(testInput.applicationFormCurrentAddresPostalCode);
+
+    expect(applicationFormPage.currentAddressStreet.getAttribute("value")).toBe(
+      testInput.applicationFormCurrentAddressStreet
+    );
+
+    expect(
+      applicationFormPage.currentAddressProvince.getAttribute("value")
+    ).toBe(testInput.applicationFormCurrentAddressProvince);
+
+    applicationFormPage.currentAddressNotSameAsMailingAddressCheckBox.click();
+    applicationFormPage.cityAndCountryBirth.sendKeys(
+      testInput.applicationFormCityAndCountryBirth
+    );
+
+    applicationFormPage.phoneNumber.sendKeys(
+      testInput.applicationFormPhoneNumber
+    );
+
+    applicationFormPage.emailAddress.sendKeys(
+      testInput.applicationFormEmailAddress
+    );
+
+    applicationFormPage.applicantPosition.sendKeys(
+      testInput.applicationFormApplicantPosition
+    );
+
+    applicationFormPage.organizationFacility.sendKeys(
+      testInput.applicationFormOrganizationFacility
+    );
+
+    applicationFormPage.mailingAddressStreet.sendKeys(
+      testInput.applicationFormMailingAddressStreet
+    );
+
+    applicationFormPage.mailingAddressCity.sendKeys(
+      testInput.applicationFormCurrentAddressCity
+    );
+
+    applicationFormPage.mailingAddressProvince.sendKeys(
+      testInput.applicationFormCurrentAddressProvince
+    );
+
+    applicationFormPage.mailingAddresPostalCode.sendKeys(
+      testInput.applicationFormCurrentAddresPostalCode
+    );
+
+    applicationFormPage.mailingAddresCountry.sendKeys(
+      testInput.applicationFormMailingAddresCountry
+    );
+
+    applicationFormPage.continueButton.click();
+    informationReviewPage.cityAndCountryBirth
+      .getText()
+      .then(function(cityAndCountryBirth) {
```

Comment:
why not use arrow functions here and all other places below?
```suggestion
      .then(cityAndCountryBirth => {
```